### ISSUE TITLE
Plot seems weird

### ISSUE THREAD

**Issue by guin0x** [wavespectra/wavespectra#70]
Hi, first of all I'd like to say sorry that I am posting this here, but I couldn't think of a more appropriate place.
I thought of stackoverflow but the chances that someone who works with `wavespectra` to read and help there would be very small. I've asked [something](https://stackoverflow.com/questions/72881297/xarray-valueerror-freq-dir-must-be-a-permuted-list-of-time-freq?noredirect=1#comment128738516_72881297) there, and someone helped me - but he also advised me to contact the developers for problems like this.

I have downloaded realtime files from the ndbc buoy 46069 and tried to use the `wavespectra` module to plot it, but the plot doesn't look correct.

My code below:

`ds = ws.read_ndbc(['46069.data_spec','46069.swdir','46069.swdir2','46069.swr1','46069.swr2'])
ds.isel(time=1).spec.plot(kind="contourf", cmap="Accent");
`

Below the plot I get:

![output](https://user-images.githubusercontent.com/35092729/177764205-3ad5cf04-4aa4-4eba-a8e5-6bb4cba540e4.png)

**Comment by rafa-guedes** [comment]
This looks like an issue with the data rather than plotting. The ndbc sample files in `wavespectra/tests/sample_files/ndbc/` folder give the same behaviour. Something does not seem right in the [ndbc reader](https://github.com/wavespectra/wavespectra/blob/master/wavespectra/input/ndbc.py) for 2D spectra. We would welcome someone reviewing that code.

**Comment by bwd007** [comment]
I am not an expert, but I have spent a bunch of hours trying to solve this issue. To me it seems like the values in the swr1 and swr2 files should be no longer scaled by 0.01. Line 143 of ndbc.py:

 ```
specdens = construct_spectra(
            specdens,
            df_swdir.values.reshape(spshape),
            df_swdir2.values.reshape(spshape),
            0.01
            * df_swr1.values.reshape(
                spshape
            ),  # these values are stored with a factor of 100
            0.01 * df_swr2.values.reshape(spshape),
            dirs,
        )
```

**Comment by guin0x** [comment]
It seems that removing the `0.01` scaling and using a `.spden` file instead of `.spec` as detailed [here](https://wavespectra.readthedocs.io/en/latest/generated/wavespectra.read_ndbc.html?highlight=read_ndbc) indeed plots something... but if whatever is being plotted is correct or not, I'm not sure. Also I don't know where to download the `.spden` files from in the NOAA website.

**Comment by bwd007** [comment]
I removed the 0.01 scaling for both swr1 and swr2 and downloaded files from https://www.ndbc.noaa.gov/data/realtime2/

```
buoy = '46002'
fnames = [buoy + '.data_spec', buoy + '.swdir', buoy + '.swdir2', buoy + '.swr1', buoy + '.swr2']
```

The results:
https://windisgood.com/waves.html#wor

**Comment by rafa-guedes** [comment]
Thanks @guin0x , @bwd007 , that seems to have solved the issue indeed. I have applied the changes in the branch https://github.com/wavespectra/wavespectra/tree/nc_ndbc/tests/sample_files/ndbc where reader for NDBC netcdf files have also been defined.

In this branch I have replaced the previous `read_ndbc` reader function by `read_ndbc_ascii`, `read_ndbc` is now a function to read NDBC data in netcdf format and could be used directly with NOAA's thredds URL. I saw you seem to be using wavespectra in your app @bwd007 (which looks great by the way) - is this change going to be a problem? Are you able to make the modifications in your app before upgrading to use the next wavespectra release?
